client/src/hooks/usePathname.tsx

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from "react";
+import { useLocation } from "react-router-dom";
+
+/**
+ * This hook returns the current pathname.
+ * @returns the current pathname
+ */
+export function usePathname() {
```

Comment:
We use the arrow function format for functions (() => {...}).


```suggestion
export const usePathname = () => {
```